flink-ml-lib/src/main/java/org/apache/flink/ml/common/datastream/EndOfStreamWindows.java

```diff
@@ -37,6 +37,8 @@
 
     private static final EndOfStreamWindows INSTANCE = new EndOfStreamWindows();
 
+    private static final TimeWindow FOREVER_WINDOW = new TimeWindow(Long.MIN_VALUE, Long.MAX_VALUE);
```

Comment:
The word `forever` seems a bit `rich` and it is rare to use this word as variable name.

Can we use a more neural word such as `TIME_WINDOW_INSTANCE`? The variable name does not need to be very descriptive here since its role is obvious from its constructor parameter values.